vnext/src/RNTester/AlertExampleUwp.tsx

```diff
@@ -0,0 +1,112 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Rather than writing a new test page, can't you just import the existing AlertExample page from react-native?